examples/ConversionForm/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { map } from "lodash";
+import React from "react";
+import JsonFileReader from "../../src/simularium/JsonFileReader";
+import { FileReturn } from "../../src/simularium/types";
+import { loadSimulariumFile } from "../../src/util";
+import InputSwitch from "./InputSwitch";
+
+interface InputFormProps {
+    template: { [key: string]: any };
+    templateData: { [key: string]: any };
+    type: string;
+    trajectory: string;
+    loadFile: (file, fileName, geoAssets?) => Promise<FileReturn | void>;
+    onReturned: () => void;
+}
+
+class InputForm extends React.Component<InputFormProps> {
+    constructor(props: InputFormProps) {
+        super(props);
+        this.state = {};
+        this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleChange(path, key, value) {
```

Comment:
again would be great to have unit tests and typings for this if possible, to help understand how it's supposed to work.